Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/AzureADIdentityProtection.py

```diff
@@ -215,6 +251,96 @@ def azure_ad_identity_protection_risky_users_dismiss_command(client: AADClient,
     return client.azure_ad_identity_protection_risky_users_dismiss(**kwargs)
 
 
+def create_incidents_from_input(input: List[Dict[str, str]], last_fetch_datetime: datetime) -> \
+        Tuple[List[Dict[str, str]], datetime]:
+
+    incidents: List[Dict[str, str]] = []
+    last_fetch = last_fetch_datetime
+
+    for current_input in input:
+        # 'activityDateTime': '2021-07-15T11:02:54Z' / 'activityDateTime': '2021-07-15T11:02:54.12345Z'
+        activity_date_time_str: str = current_input.get('activityDateTime', '')
+
+        activity_datetime = dateparser.parse(activity_date_time_str)
+        # to prevent duplicates, adding incidents with creation_time > last fetched incident
+        if last_fetch:
+            if activity_datetime <= last_fetch:
+                demisto.debug(f'{INTEGRATION_NAME} - alert {str(current_input)} created at {activity_date_time_str}.'
+                              f' Skipping.')
+                continue
+
+        current_id: str = current_input.get('id', '')
+        current_risk_event_type: str = current_input.get('riskEventType', '')
+        current_risk_detail: str = current_input.get('riskDetail', '')
+        incident = {
+            'name': f'{INTEGRATION_NAME} Incident:'
+                    f' {current_risk_event_type} {current_risk_detail} {current_id}',
```

Comment:
The incident name is very descriptive, but I'm afraid it's a bit long because the `INTEGRATION_NAME` is long. @yuvalbenshalom wdyt?